[2022-06-02 16:31:03,339] {processor.py:153} INFO - Started process (PID=47) to work on /opt/airflow/dags/test/foody/spiders/__init__.py
[2022-06-02 16:31:03,340] {processor.py:641} INFO - Processing file /opt/airflow/dags/test/foody/spiders/__init__.py for tasks to queue
[2022-06-02 16:31:03,341] {logging_mixin.py:115} INFO - [2022-06-02 16:31:03,341] {dagbag.py:507} INFO - Filling up the DagBag from /opt/airflow/dags/test/foody/spiders/__init__.py
[2022-06-02 16:31:04,781] {logging_mixin.py:115} INFO - [2022-06-02 16:31:04,781] {dagbag.py:320} ERROR - Failed to import: /opt/airflow/dags/test/foody/spiders/__init__.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 317, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/test/foody/spiders/__init__.py", line 14, in <module>
    from foody.items import FoodyItem
ModuleNotFoundError: No module named 'foody'
[2022-06-02 16:31:04,781] {processor.py:653} WARNING - No viable dags retrieved from /opt/airflow/dags/test/foody/spiders/__init__.py
[2022-06-02 16:31:04,805] {processor.py:161} INFO - Processing /opt/airflow/dags/test/foody/spiders/__init__.py took 1.471 seconds
[2022-06-02 16:31:35,586] {processor.py:153} INFO - Started process (PID=79) to work on /opt/airflow/dags/test/foody/spiders/__init__.py
[2022-06-02 16:31:35,588] {processor.py:641} INFO - Processing file /opt/airflow/dags/test/foody/spiders/__init__.py for tasks to queue
[2022-06-02 16:31:35,588] {logging_mixin.py:115} INFO - [2022-06-02 16:31:35,588] {dagbag.py:507} INFO - Filling up the DagBag from /opt/airflow/dags/test/foody/spiders/__init__.py
[2022-06-02 16:31:36,444] {logging_mixin.py:115} INFO - [2022-06-02 16:31:36,443] {dagbag.py:320} ERROR - Failed to import: /opt/airflow/dags/test/foody/spiders/__init__.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 317, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/test/foody/spiders/__init__.py", line 14, in <module>
    from foody.items import FoodyItem
ModuleNotFoundError: No module named 'foody'
[2022-06-02 16:31:36,444] {processor.py:653} WARNING - No viable dags retrieved from /opt/airflow/dags/test/foody/spiders/__init__.py
[2022-06-02 16:31:36,459] {processor.py:161} INFO - Processing /opt/airflow/dags/test/foody/spiders/__init__.py took 0.878 seconds
[2022-06-02 16:32:06,719] {processor.py:153} INFO - Started process (PID=100) to work on /opt/airflow/dags/test/foody/spiders/__init__.py
[2022-06-02 16:32:06,720] {processor.py:641} INFO - Processing file /opt/airflow/dags/test/foody/spiders/__init__.py for tasks to queue
[2022-06-02 16:32:06,720] {logging_mixin.py:115} INFO - [2022-06-02 16:32:06,720] {dagbag.py:507} INFO - Filling up the DagBag from /opt/airflow/dags/test/foody/spiders/__init__.py
[2022-06-02 16:32:07,629] {logging_mixin.py:115} INFO - [2022-06-02 16:32:07,629] {dagbag.py:320} ERROR - Failed to import: /opt/airflow/dags/test/foody/spiders/__init__.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 317, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/test/foody/spiders/__init__.py", line 14, in <module>
    from foody.items import FoodyItem
ModuleNotFoundError: No module named 'foody'
[2022-06-02 16:32:07,629] {processor.py:653} WARNING - No viable dags retrieved from /opt/airflow/dags/test/foody/spiders/__init__.py
[2022-06-02 16:32:07,645] {processor.py:161} INFO - Processing /opt/airflow/dags/test/foody/spiders/__init__.py took 0.930 seconds
[2022-06-02 16:32:37,819] {processor.py:153} INFO - Started process (PID=130) to work on /opt/airflow/dags/test/foody/spiders/__init__.py
[2022-06-02 16:32:37,820] {processor.py:641} INFO - Processing file /opt/airflow/dags/test/foody/spiders/__init__.py for tasks to queue
[2022-06-02 16:32:37,820] {logging_mixin.py:115} INFO - [2022-06-02 16:32:37,820] {dagbag.py:507} INFO - Filling up the DagBag from /opt/airflow/dags/test/foody/spiders/__init__.py
[2022-06-02 16:32:38,728] {logging_mixin.py:115} INFO - [2022-06-02 16:32:38,727] {dagbag.py:320} ERROR - Failed to import: /opt/airflow/dags/test/foody/spiders/__init__.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 317, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/test/foody/spiders/__init__.py", line 14, in <module>
    from foody.items import FoodyItem
ModuleNotFoundError: No module named 'foody'
[2022-06-02 16:32:38,728] {processor.py:653} WARNING - No viable dags retrieved from /opt/airflow/dags/test/foody/spiders/__init__.py
[2022-06-02 16:32:38,745] {processor.py:161} INFO - Processing /opt/airflow/dags/test/foody/spiders/__init__.py took 0.928 seconds
[2022-06-02 16:33:08,924] {processor.py:153} INFO - Started process (PID=160) to work on /opt/airflow/dags/test/foody/spiders/__init__.py
[2022-06-02 16:33:08,925] {processor.py:641} INFO - Processing file /opt/airflow/dags/test/foody/spiders/__init__.py for tasks to queue
[2022-06-02 16:33:08,925] {logging_mixin.py:115} INFO - [2022-06-02 16:33:08,925] {dagbag.py:507} INFO - Filling up the DagBag from /opt/airflow/dags/test/foody/spiders/__init__.py
[2022-06-02 16:33:09,921] {logging_mixin.py:115} INFO - [2022-06-02 16:33:09,920] {dagbag.py:320} ERROR - Failed to import: /opt/airflow/dags/test/foody/spiders/__init__.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 317, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/test/foody/spiders/__init__.py", line 14, in <module>
    from foody.items import FoodyItem
ModuleNotFoundError: No module named 'foody'
[2022-06-02 16:33:09,921] {processor.py:653} WARNING - No viable dags retrieved from /opt/airflow/dags/test/foody/spiders/__init__.py
[2022-06-02 16:33:09,943] {processor.py:161} INFO - Processing /opt/airflow/dags/test/foody/spiders/__init__.py took 1.022 seconds
[2022-06-02 16:33:40,019] {processor.py:153} INFO - Started process (PID=181) to work on /opt/airflow/dags/test/foody/spiders/__init__.py
[2022-06-02 16:33:40,020] {processor.py:641} INFO - Processing file /opt/airflow/dags/test/foody/spiders/__init__.py for tasks to queue
[2022-06-02 16:33:40,021] {logging_mixin.py:115} INFO - [2022-06-02 16:33:40,020] {dagbag.py:507} INFO - Filling up the DagBag from /opt/airflow/dags/test/foody/spiders/__init__.py
[2022-06-02 16:33:40,924] {logging_mixin.py:115} INFO - [2022-06-02 16:33:40,924] {dagbag.py:320} ERROR - Failed to import: /opt/airflow/dags/test/foody/spiders/__init__.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 317, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/test/foody/spiders/__init__.py", line 14, in <module>
    from foody.items import FoodyItem
ModuleNotFoundError: No module named 'foody'
[2022-06-02 16:33:40,924] {processor.py:653} WARNING - No viable dags retrieved from /opt/airflow/dags/test/foody/spiders/__init__.py
[2022-06-02 16:33:40,942] {processor.py:161} INFO - Processing /opt/airflow/dags/test/foody/spiders/__init__.py took 0.926 seconds
[2022-06-02 16:34:11,137] {processor.py:153} INFO - Started process (PID=211) to work on /opt/airflow/dags/test/foody/spiders/__init__.py
[2022-06-02 16:34:11,138] {processor.py:641} INFO - Processing file /opt/airflow/dags/test/foody/spiders/__init__.py for tasks to queue
[2022-06-02 16:34:11,139] {logging_mixin.py:115} INFO - [2022-06-02 16:34:11,139] {dagbag.py:507} INFO - Filling up the DagBag from /opt/airflow/dags/test/foody/spiders/__init__.py
[2022-06-02 16:34:11,959] {logging_mixin.py:115} INFO - [2022-06-02 16:34:11,959] {dagbag.py:320} ERROR - Failed to import: /opt/airflow/dags/test/foody/spiders/__init__.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 317, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/test/foody/spiders/__init__.py", line 14, in <module>
    from foody.items import FoodyItem
ModuleNotFoundError: No module named 'foody'
[2022-06-02 16:34:11,960] {processor.py:653} WARNING - No viable dags retrieved from /opt/airflow/dags/test/foody/spiders/__init__.py
[2022-06-02 16:34:12,013] {processor.py:161} INFO - Processing /opt/airflow/dags/test/foody/spiders/__init__.py took 0.879 seconds
[2022-06-02 16:34:42,294] {processor.py:153} INFO - Started process (PID=231) to work on /opt/airflow/dags/test/foody/spiders/__init__.py
[2022-06-02 16:34:42,295] {processor.py:641} INFO - Processing file /opt/airflow/dags/test/foody/spiders/__init__.py for tasks to queue
[2022-06-02 16:34:42,295] {logging_mixin.py:115} INFO - [2022-06-02 16:34:42,295] {dagbag.py:507} INFO - Filling up the DagBag from /opt/airflow/dags/test/foody/spiders/__init__.py
[2022-06-02 16:34:43,243] {logging_mixin.py:115} INFO - [2022-06-02 16:34:43,243] {dagbag.py:320} ERROR - Failed to import: /opt/airflow/dags/test/foody/spiders/__init__.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 317, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/test/foody/spiders/__init__.py", line 14, in <module>
    from foody.items import FoodyItem
ModuleNotFoundError: No module named 'foody'
[2022-06-02 16:34:43,243] {processor.py:653} WARNING - No viable dags retrieved from /opt/airflow/dags/test/foody/spiders/__init__.py
[2022-06-02 16:34:43,260] {processor.py:161} INFO - Processing /opt/airflow/dags/test/foody/spiders/__init__.py took 0.970 seconds
[2022-06-02 16:35:13,386] {processor.py:153} INFO - Started process (PID=261) to work on /opt/airflow/dags/test/foody/spiders/__init__.py
[2022-06-02 16:35:13,387] {processor.py:641} INFO - Processing file /opt/airflow/dags/test/foody/spiders/__init__.py for tasks to queue
[2022-06-02 16:35:13,387] {logging_mixin.py:115} INFO - [2022-06-02 16:35:13,387] {dagbag.py:507} INFO - Filling up the DagBag from /opt/airflow/dags/test/foody/spiders/__init__.py
[2022-06-02 16:35:14,348] {logging_mixin.py:115} INFO - [2022-06-02 16:35:14,347] {dagbag.py:320} ERROR - Failed to import: /opt/airflow/dags/test/foody/spiders/__init__.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 317, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/test/foody/spiders/__init__.py", line 14, in <module>
    from foody.items import FoodyItem
ModuleNotFoundError: No module named 'foody'
[2022-06-02 16:35:14,348] {processor.py:653} WARNING - No viable dags retrieved from /opt/airflow/dags/test/foody/spiders/__init__.py
[2022-06-02 16:35:14,364] {processor.py:161} INFO - Processing /opt/airflow/dags/test/foody/spiders/__init__.py took 0.981 seconds
[2022-06-02 16:35:44,472] {processor.py:153} INFO - Started process (PID=291) to work on /opt/airflow/dags/test/foody/spiders/__init__.py
[2022-06-02 16:35:44,474] {processor.py:641} INFO - Processing file /opt/airflow/dags/test/foody/spiders/__init__.py for tasks to queue
[2022-06-02 16:35:44,475] {logging_mixin.py:115} INFO - [2022-06-02 16:35:44,474] {dagbag.py:507} INFO - Filling up the DagBag from /opt/airflow/dags/test/foody/spiders/__init__.py
[2022-06-02 16:35:45,386] {logging_mixin.py:115} INFO - [2022-06-02 16:35:45,386] {dagbag.py:320} ERROR - Failed to import: /opt/airflow/dags/test/foody/spiders/__init__.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 317, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/test/foody/spiders/__init__.py", line 14, in <module>
    from foody.items import FoodyItem
ModuleNotFoundError: No module named 'foody'
[2022-06-02 16:35:45,386] {processor.py:653} WARNING - No viable dags retrieved from /opt/airflow/dags/test/foody/spiders/__init__.py
[2022-06-02 16:35:45,403] {processor.py:161} INFO - Processing /opt/airflow/dags/test/foody/spiders/__init__.py took 0.936 seconds
[2022-06-02 16:36:15,573] {processor.py:153} INFO - Started process (PID=312) to work on /opt/airflow/dags/test/foody/spiders/__init__.py
[2022-06-02 16:36:15,574] {processor.py:641} INFO - Processing file /opt/airflow/dags/test/foody/spiders/__init__.py for tasks to queue
[2022-06-02 16:36:15,575] {logging_mixin.py:115} INFO - [2022-06-02 16:36:15,575] {dagbag.py:507} INFO - Filling up the DagBag from /opt/airflow/dags/test/foody/spiders/__init__.py
[2022-06-02 16:36:16,394] {logging_mixin.py:115} INFO - [2022-06-02 16:36:16,394] {dagbag.py:320} ERROR - Failed to import: /opt/airflow/dags/test/foody/spiders/__init__.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 317, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/test/foody/spiders/__init__.py", line 14, in <module>
    from foody.items import FoodyItem
ModuleNotFoundError: No module named 'foody'
[2022-06-02 16:36:16,395] {processor.py:653} WARNING - No viable dags retrieved from /opt/airflow/dags/test/foody/spiders/__init__.py
[2022-06-02 16:36:16,411] {processor.py:161} INFO - Processing /opt/airflow/dags/test/foody/spiders/__init__.py took 0.841 seconds
[2022-06-02 16:36:46,729] {processor.py:153} INFO - Started process (PID=344) to work on /opt/airflow/dags/test/foody/spiders/__init__.py
[2022-06-02 16:36:46,731] {processor.py:641} INFO - Processing file /opt/airflow/dags/test/foody/spiders/__init__.py for tasks to queue
[2022-06-02 16:36:46,731] {logging_mixin.py:115} INFO - [2022-06-02 16:36:46,731] {dagbag.py:507} INFO - Filling up the DagBag from /opt/airflow/dags/test/foody/spiders/__init__.py
[2022-06-02 16:36:47,773] {logging_mixin.py:115} INFO - [2022-06-02 16:36:47,773] {dagbag.py:320} ERROR - Failed to import: /opt/airflow/dags/test/foody/spiders/__init__.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 317, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/test/foody/spiders/__init__.py", line 14, in <module>
    from foody.items import FoodyItem
ModuleNotFoundError: No module named 'foody'
[2022-06-02 16:36:47,773] {processor.py:653} WARNING - No viable dags retrieved from /opt/airflow/dags/test/foody/spiders/__init__.py
[2022-06-02 16:36:47,790] {processor.py:161} INFO - Processing /opt/airflow/dags/test/foody/spiders/__init__.py took 1.065 seconds
[2022-06-02 16:37:17,879] {processor.py:153} INFO - Started process (PID=375) to work on /opt/airflow/dags/test/foody/spiders/__init__.py
[2022-06-02 16:37:17,880] {processor.py:641} INFO - Processing file /opt/airflow/dags/test/foody/spiders/__init__.py for tasks to queue
[2022-06-02 16:37:17,880] {logging_mixin.py:115} INFO - [2022-06-02 16:37:17,880] {dagbag.py:507} INFO - Filling up the DagBag from /opt/airflow/dags/test/foody/spiders/__init__.py
[2022-06-02 16:37:18,811] {logging_mixin.py:115} INFO - [2022-06-02 16:37:18,811] {dagbag.py:320} ERROR - Failed to import: /opt/airflow/dags/test/foody/spiders/__init__.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 317, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/test/foody/spiders/__init__.py", line 14, in <module>
    from foody.items import FoodyItem
ModuleNotFoundError: No module named 'foody'
[2022-06-02 16:37:18,812] {processor.py:653} WARNING - No viable dags retrieved from /opt/airflow/dags/test/foody/spiders/__init__.py
[2022-06-02 16:37:18,829] {processor.py:161} INFO - Processing /opt/airflow/dags/test/foody/spiders/__init__.py took 0.954 seconds
[2022-06-02 16:37:48,942] {processor.py:153} INFO - Started process (PID=398) to work on /opt/airflow/dags/test/foody/spiders/__init__.py
[2022-06-02 16:37:48,943] {processor.py:641} INFO - Processing file /opt/airflow/dags/test/foody/spiders/__init__.py for tasks to queue
[2022-06-02 16:37:48,943] {logging_mixin.py:115} INFO - [2022-06-02 16:37:48,943] {dagbag.py:507} INFO - Filling up the DagBag from /opt/airflow/dags/test/foody/spiders/__init__.py
[2022-06-02 16:37:49,889] {logging_mixin.py:115} INFO - [2022-06-02 16:37:49,889] {dagbag.py:320} ERROR - Failed to import: /opt/airflow/dags/test/foody/spiders/__init__.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 317, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/test/foody/spiders/__init__.py", line 14, in <module>
    from foody.items import FoodyItem
ModuleNotFoundError: No module named 'foody'
[2022-06-02 16:37:49,889] {processor.py:653} WARNING - No viable dags retrieved from /opt/airflow/dags/test/foody/spiders/__init__.py
[2022-06-02 16:37:49,913] {processor.py:161} INFO - Processing /opt/airflow/dags/test/foody/spiders/__init__.py took 0.974 seconds
[2022-06-02 16:38:20,058] {processor.py:153} INFO - Started process (PID=429) to work on /opt/airflow/dags/test/foody/spiders/__init__.py
[2022-06-02 16:38:20,059] {processor.py:641} INFO - Processing file /opt/airflow/dags/test/foody/spiders/__init__.py for tasks to queue
[2022-06-02 16:38:20,060] {logging_mixin.py:115} INFO - [2022-06-02 16:38:20,059] {dagbag.py:507} INFO - Filling up the DagBag from /opt/airflow/dags/test/foody/spiders/__init__.py
[2022-06-02 16:38:21,080] {logging_mixin.py:115} INFO - [2022-06-02 16:38:21,080] {dagbag.py:320} ERROR - Failed to import: /opt/airflow/dags/test/foody/spiders/__init__.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 317, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/test/foody/spiders/__init__.py", line 14, in <module>
    from foody.items import FoodyItem
ModuleNotFoundError: No module named 'foody'
[2022-06-02 16:38:21,080] {processor.py:653} WARNING - No viable dags retrieved from /opt/airflow/dags/test/foody/spiders/__init__.py
[2022-06-02 16:38:21,098] {processor.py:161} INFO - Processing /opt/airflow/dags/test/foody/spiders/__init__.py took 1.047 seconds
[2022-06-02 16:41:16,497] {processor.py:153} INFO - Started process (PID=48) to work on /opt/airflow/dags/test/foody/spiders/__init__.py
[2022-06-02 16:41:16,498] {processor.py:641} INFO - Processing file /opt/airflow/dags/test/foody/spiders/__init__.py for tasks to queue
[2022-06-02 16:41:16,499] {logging_mixin.py:115} INFO - [2022-06-02 16:41:16,499] {dagbag.py:507} INFO - Filling up the DagBag from /opt/airflow/dags/test/foody/spiders/__init__.py
[2022-06-02 16:41:17,967] {logging_mixin.py:115} INFO - [2022-06-02 16:41:17,967] {dagbag.py:320} ERROR - Failed to import: /opt/airflow/dags/test/foody/spiders/__init__.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 317, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/test/foody/spiders/__init__.py", line 14, in <module>
    from foody.items import FoodyItem
ModuleNotFoundError: No module named 'foody'
[2022-06-02 16:41:17,968] {processor.py:653} WARNING - No viable dags retrieved from /opt/airflow/dags/test/foody/spiders/__init__.py
[2022-06-02 16:41:17,989] {processor.py:161} INFO - Processing /opt/airflow/dags/test/foody/spiders/__init__.py took 1.496 seconds
[2022-06-02 16:41:48,654] {processor.py:153} INFO - Started process (PID=79) to work on /opt/airflow/dags/test/foody/spiders/__init__.py
[2022-06-02 16:41:48,655] {processor.py:641} INFO - Processing file /opt/airflow/dags/test/foody/spiders/__init__.py for tasks to queue
[2022-06-02 16:41:48,655] {logging_mixin.py:115} INFO - [2022-06-02 16:41:48,655] {dagbag.py:507} INFO - Filling up the DagBag from /opt/airflow/dags/test/foody/spiders/__init__.py
[2022-06-02 16:41:49,568] {logging_mixin.py:115} INFO - [2022-06-02 16:41:49,568] {dagbag.py:320} ERROR - Failed to import: /opt/airflow/dags/test/foody/spiders/__init__.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 317, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/test/foody/spiders/__init__.py", line 14, in <module>
    from foody.items import FoodyItem
ModuleNotFoundError: No module named 'foody'
[2022-06-02 16:41:49,569] {processor.py:653} WARNING - No viable dags retrieved from /opt/airflow/dags/test/foody/spiders/__init__.py
[2022-06-02 16:41:49,584] {processor.py:161} INFO - Processing /opt/airflow/dags/test/foody/spiders/__init__.py took 0.933 seconds
[2022-06-02 16:42:19,798] {processor.py:153} INFO - Started process (PID=100) to work on /opt/airflow/dags/test/foody/spiders/__init__.py
[2022-06-02 16:42:19,800] {processor.py:641} INFO - Processing file /opt/airflow/dags/test/foody/spiders/__init__.py for tasks to queue
[2022-06-02 16:42:19,802] {logging_mixin.py:115} INFO - [2022-06-02 16:42:19,802] {dagbag.py:507} INFO - Filling up the DagBag from /opt/airflow/dags/test/foody/spiders/__init__.py
[2022-06-02 16:42:20,838] {logging_mixin.py:115} INFO - [2022-06-02 16:42:20,838] {dagbag.py:320} ERROR - Failed to import: /opt/airflow/dags/test/foody/spiders/__init__.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 317, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/test/foody/spiders/__init__.py", line 14, in <module>
    from foody.items import FoodyItem
ModuleNotFoundError: No module named 'foody'
[2022-06-02 16:42:20,838] {processor.py:653} WARNING - No viable dags retrieved from /opt/airflow/dags/test/foody/spiders/__init__.py
[2022-06-02 16:42:20,855] {processor.py:161} INFO - Processing /opt/airflow/dags/test/foody/spiders/__init__.py took 1.068 seconds
